test/kubemark/start-kubemark.sh

```diff
@@ -22,36 +22,41 @@ KUBE_ROOT=$(readlink -e ${TMP_ROOT} 2> /dev/null || perl -MCwd -e 'print Cwd::ab
 
 source "${KUBE_ROOT}/test/kubemark/common.sh"
 
-function writeEnvironmentFiles() {
-  cat > "${RESOURCE_DIRECTORY}/apiserver_flags" <<EOF
-${APISERVER_TEST_ARGS}
---storage-backend=${STORAGE_BACKEND}
---service-cluster-ip-range="${SERVICE_CLUSTER_IP_RANGE}"
+function writeEnvironmentFile() {
+  cat > "${RESOURCE_DIRECTORY}/kubemark-master-env.sh" <<EOF
+# Generic variables.
+INSTANCE_PREFIX=${INSTANCE_PREFIX}
+SERVICE_CLUSTER_IP_RANGE=${SERVICE_CLUSTER_IP_RANGE}
+
+# Kubelet related variables.
+MASTER_IP_RANGE=${MASTER_IP_RANGE}
```

Comment:
This is used only in standalone-kubelet mode. We are not using it in kubemark at all.